### PR TITLE
feat: adding the wcag wcag131 tag to the aria hidden body rule

### LINKED ISSUE #4315: Have the aria-hidden-body rule map to 1.3.1
The `aria-hidden-body` rule has the `wcag412` tag, that's probably correct, but it might also be good to add `wcag131` to the list. That's just as likely to be failed when `aria-hidden-body` finds a problem. This can be fixed by adding a tag to the `aria-hidden-body` rule.

### PR DESCRIPTION
**Pull request by CBID2**
<< Describe the changes >>
This PR adds the `wcag412` tag to the aria-hidden-body rule, making it easier for the testing engine to detect issues.

Closes #4315

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @straker! :) My PR is ready for review! :)

File: lib/rules/aria-hidden-body.json [hunk 1]
```diff
@@ -4,7 +4,14 @@
...
   "excludeHidden": false,
   "matches": "is-initiator-matches",
-  "tags": ["cat.aria", "wcag2a", "wcag412", "EN-301-549", "EN-9.4.1.2"],
+  "tags": [
+    "cat.aria",
+    "wcag2a",
+    "wcag131",
+    "wcag412",
+    "EN-301-549",
+    "EN-9.4.1.2"
```
Comment (WilcoFiers) [inline lib/rules/aria-hidden-body.json:13]:
```suggestion
    "EN-9.1.3.1",
    "EN-9.4.1.2"
```

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
Build is failing because of this. Also please update the PR title, you're adding wcag131, not 412.

**Comment by CBID2** [comment]
> wcag131

Done @WilcoFiers

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
Reviewed for security.

**Pull request merged**